doc/acceleration-support.md

```diff
@@ -0,0 +1,117 @@
+# Acceleration Support
+
+The `ggml-sys` crate includes various acceleration backends, selectable via `--features` flags. The availability of supported backends varies by platform, and `ggml-sys` can only be built with a single active acceleration backend at a time. If cublas and clblast are both specified, cublas is prioritized and clblast is ignored.
```

Comment:
Also applies to `llm` etc - maybe say "The `llm` ecosystem of crates, including `llm`, `llm-base` and `ggml` support various acceleration backends"

Reply:
CuBLAS and CLBlast (consistent capitalisation is important when referring to the solutions and not the features)